scripts/remove_artifacts.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import os
+import sys
+import time
+import yaml
+from tqdm import tqdm
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import rasterio
+
+sys.path.insert(0, '.')
+import functions.metrics as metrics
+import functions.misc as misc
+from functions.constants import DONE_MSG
+
+from loguru import logger
+logger = misc.format_logger(logger)
+
+
+if __name__ == "__main__":
+
+    # Chronometer
+    tic = time.time()
+    logger.info('Starting...')
+
+    # Argument and parameter specification
+    parser = argparse.ArgumentParser(description="The script removes artifacts by comparing dets and tiles.")
```

Comment:
Give a better description of what the script is doing: 
"The script removes artifacts (probable false detections similar to tiles) by comparing dets and tiles."